### DISCUSSION TITLE
Going inactive for 2 years as I'll be preparing for my entrance exams

### DISCUSSION THREAD

**Discussion by thunder-coding** [termux/termux-packages#11221] [Announcements]
Everyone has a dream college, and even I have one. But for studying in my dream college I'll have to compete with 12 lakh other aspirants who are looking forward for the same seat. And hence I'll have to prepare hard, take inspirations from every small achievements and work forward to join all the small pieces of the map.

Hence I announce that for the next 2 years, I'll be **highly** inactive here. I want to concentrate on my studies and rise up.

I'm also ready to give up maintenance of nodejs and nodejs-lts to anyone else who's interested and proves to be someone who can maintain the packages properly with responsibility. 

I'll continue to be a member of the organisation and also help someone when they explicitly ping me for help. In case you want a reply from me, you'll have to ping me for that.

Hope that you guys will understand and help me isolate myself from most social media stuff. With hopes that with your blessings and good wishes, I'll make it!

With regards,
Yaksh.

**Comment by kcubeterm** [comment]
Good luck for your entrance exam. I'm sure you will be in your dream college. Don't forget to eat and sleep.

**Reply by thunder-coding** [reply]
Sure, thanks for reminding

**Comment by T-Dynamos** [comment]
Good luck !. Can I know your dream college?
and we will see you at 12:00am , Wednesday 10 July , 2024

**Reply by kcubeterm** [reply]
Let's guess!! He is good at mathematics. It means he must not be a medical student. Since he could do programming stuff as well. So we can assume, he may go for computer science. 

IIT Bombay is one of the best college for computer science in the world. Also it is near to his home. It must be IIT Bombay.

**Comment by Grimler91** [comment]
Good luck, I am sure you will do great :)

**Comment by licy183** [comment]
Good luck! I wish you success!

**Comment by 2096779623** [comment]
Good luck! I guess your dream is to go to Indian Institute of Technology?

**Comment by DreamOneX** [comment]
Good luck !

**Comment by thunder-coding** [comment]
Thanks everyone for the wishes. It'll really motivate me to stick to my goal and land myself in my dream college in 2 years.

And yes it's IIT as many of you guys pointed out. (IIT-B to be more specific as it's both near 9t my residence and one of the best colleges in India).

And again thanks for the good wishes!

**Reply by finagolfin** [reply]
I suggest you don't spend _so_ much time on getting into college, as college is largely a scam. Instead, you now have a world of information at your fingertips online, you are much better off following your own interests and reading and learning online. I joined one of the top engineering colleges 25 years ago, and I have seen nothing since to convince me otherwise. This has nothing to do with your stopping working on Termux, which you are not going to learn that much by doing either.

Instead, if you want to learn CS or any other technical discipline, I encourage you to learn by doing, eg buy some lisp, C#, or Haskell books and work through them with a compiler yourself. If you ever get stuck or don't know what to do, my email is public and many other knowledgeable people are an email or IM away: ask questions. This is how most people learned for most of history, college usually doesn't work better, often it is a lot worse.

I suspect you will have a much better life if you do this rather than wasting your time preparing for a college exam. I know you didn't ask for this advice, but it drives me nuts how colleges keep scamming young kids like you and especially your parents.

**Reply by kcubeterm** [reply]
Thanks for your precious advice. this is what two year back, a legend told me. Your public email is appreciated. never knew you are that senior. 

>  I know you didn't ask for this advice

Who don't want to consider an advice which is coming from an experienced person. its should be your duty to pass your knowledge,experience, advice to successor generation. one of the reason why we human are best among all animals.

**Reply by T-Dynamos** [reply]
> Thanks for your precious advice. this is what two year back, a legend told me. Your public email is appreciated. never knew you are that senior.
> 
> > I know you didn't ask for this advice
> 
> Who don't want to consider an advice which is coming from an experienced person. its should be your duty to pass your knowledge,experience, advice to successor generation. one of the reason why we human are best among all animals.

Problem is that who is going to explain this to our _Indian_ parents. @thunder-coding will understand this better (because he's also indian)

**Reply by kcubeterm** [reply]
> Problem is that who is going to explain this to our Indian parents

Seems like @buttaface  is ready to take risk.

**Comment by hgaddy2** [comment]
Good luck, buddy. Your contributions are much appreciated. See you on the other side.

**Comment by Venom243** [comment]
Good luck tho

**Comment by Exanoff12** [comment]
Best of luck !

**Comment by ghost** [comment]
Good luck buddy! Best wishes on your journey!
